examples/with_generate_docs.ts

```diff
@@ -0,0 +1,5 @@
+//deno:generate -command gen_docs deno run -A generate_docs.ts
```

Comment:
typically named parameters use `--` as a prefix, to disambiguate them from parameters that are a combination of one-letter parameters (ex: `tar -cvf`)

Reply:
> typically named parameters use `--` as a prefix, to disambiguate them from parameters that are a combination of one-letter parameters (ex: `tar -cvf`)

The `-command` alias in `//go:generate -command` is a way to create an alias for a command that you want to run later. It allows you to define the command once and then use the alias multiple times in the file it was defined in, appending different arguments to the alias command when you need to run it. The `//deno:generate -command` directive is inspired by  `//go:generate -command`.